### DISCUSSION TITLE
[1차 프로젝트 제출 보고서 모음(1차 기획 보고서/ UI / DB / 프로젝트 구조 설계 보고서)]

### DISCUSSION THREAD

**Discussion by uforset** [uforset/git_test#4] [Announcements]
1. [<1차 프로젝트> 기획 보고서.pdf](https://github.com/uforset/project_1st/files/9158231/3._1._.1.pdf)
2. [<1차 프로젝트> UI 설계 보고서.pdf](https://github.com/uforset/project_1st/files/9158234/1.UI.pdf)
3. [<1차 프로젝트> DB 설계 보고서.pdf](https://github.com/uforset/project_1st/files/9158235/1.DB.pdf)
4. [<1차 프로젝트> 프로젝트 구조 설계 보고서.pdf](https://github.com/uforset/project_1st/files/9158236/1.pdf)

**Comment by heyyeong** [comment]
감사합니다!
